Adding or removing dependency breaks iOS build

Hello there, first of all, thank you so much for your work on this package. I had a really hard time figuring why my emojis were breaking when I tried to split and your effort helped me greatly in understand what was going on.

The reason I stumbled across your package was because I was working on a simple that added spaces around characters, when I noticed that some emojis broke it I did the following:
- Modified my `pubspec.yaml` file to include the new dependency
- Ran `flutter pub get`
- Opened IntelliJ and try to run the app using the simulator

However I got an error while trying to execute the app, this is the stack trace:
```
Launching lib/main.dart on iPhone 6 in debug mode...
Running pod install...
CocoaPods' output:
↳
      Preparing

    Analyzing dependencies

    Inspecting targets to integrate
      Using `ARCHS` setting to build architectures of target `Pods-Runner`: (``)

    Fetching external sources
    -> Fetching podspec for `Flutter` from `.symlinks/flutter/ios`
    -> Fetching podspec for `icu` from `.symlinks/plugins/icu/ios`

    Resolving dependencies of `Podfile`

    Comparing resolved specification to the sandbox manifest
      A Flutter
      A icu

    Downloading dependencies

    -> Installing Flutter (1.0.0)

    -> Installing icu (0.0.1)
      - Running pre install hooks
    [!] Unable to determine Swift version for the following pods:

    - `icu` does not specify a Swift version and none of the targets (`Runner`) integrating it have the `SWIFT_VERSION` attribute set. Please contact the author or set the `SWIFT_VERSION` attribute in at least one of the targets that integrate this pod.

    /Users/leo/.gem/ruby/2.3.1/gems/cocoapods-1.7.5/lib/cocoapods/installer/xcode/target_validator.rb:122:in `verify_swift_pods_swift_version'
    /Users/leo/.gem/ruby/2.3.1/gems/cocoapods-1.7.5/lib/cocoapods/installer/xcode/target_validator.rb:37:in `validate!'
    /Users/leo/.gem/ruby/2.3.1/gems/cocoapods-1.7.5/lib/cocoapods/installer.rb:578:in `validate_targets'
    /Users/leo/.gem/ruby/2.3.1/gems/cocoapods-1.7.5/lib/cocoapods/installer.rb:158:in `install!'
    /Users/leo/.gem/ruby/2.3.1/gems/cocoapods-1.7.5/lib/cocoapods/command/install.rb:51:in `run'
    /Users/leo/.gem/ruby/2.3.1/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
    /Users/leo/.gem/ruby/2.3.1/gems/cocoapods-1.7.5/lib/cocoapods/command.rb:52:in `run'
    /Users/leo/.gem/ruby/2.3.1/gems/cocoapods-1.7.5/bin/pod:55:in `<top (required)>'
    /Users/leo/.gem/ruby/2.3.1/bin/pod:23:in `load'
    /Users/leo/.gem/ruby/2.3.1/bin/pod:23:in `<main>'

Error output from CocoaPods:
↳

    [!] `<PBXGroup UUID=`97C146E51CF9000F007C117D`>` attempted to initialize an object with an unknown UUID. `CF3B75C9A7D2FA2A4C99F110` for attribute: `children`. This can be the result of a merge and  the unknown UUID is being discarded.

    [!] Automatically assigning platform `ios` with version `8.0` on target `Runner` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.

Error running pod install
Error launching application on iPhone 6.

```

I then proceeded to remove the dependency from the `pubspec` file but I am still unable to run the app, it seems that during the initial setup something changed in the iOS config files that prevents the app from recovering, this is the stack trace after removing the package:
```
Launching lib/main.dart on iPhone 6 in debug mode...
Running Xcode build...
Xcode build done.                                            1.6s
Failed to build iOS app
Error output from Xcode build:
↳
    2019-09-19 18:21:33.233 xcodebuild[57818:3898200] [MT] DVTPlugInManager: Required plug-in compatibility UUID B89EAABF-783E-4EBF-80D4-A9EAC69F77F2 for KSImageNamed.ideplugin (com.ksuther.KSImageNamed) not present
    ** BUILD FAILED **


Xcode's output:
↳
    error: /Users/leo/Sites/grid-connect/flutter-tests/boyespeak/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths (in target 'Runner')
    error: /Users/leo/Sites/grid-connect/flutter-tests/boyespeak/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths (in target 'Runner')
    error: /Users/leo/Sites/grid-connect/flutter-tests/boyespeak/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths (in target 'Runner')
    error: /Users/leo/Sites/grid-connect/flutter-tests/boyespeak/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths (in target 'Runner')
    error: /Users/leo/Sites/grid-connect/flutter-tests/boyespeak/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths (in target 'Runner')
    error: /Users/leo/Sites/grid-connect/flutter-tests/boyespeak/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths (in target 'Runner')
    note: Using new build systemnote: Planning buildnote: Constructing build description

Could not build the application for the simulator.
Error launching application on iPhone 6.
```

I uploaded the code for the app along [with a commit that demonstrates the breakage over here](https://github.com/leopic/boyespeak/commit/93716ff6a976cd8976c78534b8b817fa55fcf4cf).


Thanks again!

--- a/ios/icu.podspec
+++ b/ios/icu.podspec
@@ -15,7 +15,7 @@ A new flutter plugin project.
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
- s.swift_version = "4.0"
+  s.swift_version = "4.0"
 
   s.ios.deployment_target = '8.0'
 end

--- a/ios/icu.podspec
+++ b/ios/icu.podspec
@@ -15,6 +15,7 @@ A new flutter plugin project.
   s.source_files = 'Classes/**/*'
   s.public_header_files = 'Classes/**/*.h'
   s.dependency 'Flutter'
+ s.swift_version = "4.0"
 
   s.ios.deployment_target = '8.0'
 end

